server/mlabns/conf/tools.csv

```diff
@@ -1,5 +1,6 @@
 tool_id,slice_id,http_port,show_tool_extra
 ndt,iupui_ndt,7123,False
+ndt_ssl,iupui_ndt,7123,False
```

Comment:
Oh, I see what you're saying. It doesn't make sense to have NDT-SSL advertise fakewww because it will just direct clients to NDT-plaintext. Alright, I think it makes sense to just have nothing here because if a client wants NDT-SSL and wants to get to it through a web portal, fakewww doesn't help them, so we might as well say there's no web portal for NDT-SSL at the moment. `3010` would be confusing because this is a port that we're telling clients can speak HTTP and NDT-SSL can't speak HTTP.

Fixed, good catch.